battle/hearts_container.gd

```diff
@@ -0,0 +1,32 @@
+extends HBoxContainer
+
+@onready var stats = $"../../Stats"
+@onready var HeartGuiClass = preload("res://battle/heart_gui.tscn")
+@onready var maxHP = stats.HP
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	setHearts(maxHP) # adds max hp hearts to the character
+
+func setHearts(num: int):
+	var current_hearts = get_child_count()
+	if num > current_hearts: # we have too many hearts, remove the excess
```

Comment:
For the comment for line 12 is it supposed to be that we don't have enough hearts so we need to add more?

Reply:
yes, it is I accidentally put that comment twice.